### ISSUE TITLE
Archive this repo?

### ISSUE THREAD

**Issue by chadoh** [nearprotocol/near-fiddle-api#79]
@vgrichina can we archive this repository along with nearprotocol/NEARStudio?

**Comment by vgrichina** [comment]
@chadoh yes if doesn't break existing deployment on Render (I sometimes go scavenging old code)

**Comment by chadoh** [comment]
![image](https://user-images.githubusercontent.com/221614/83190554-97f96f80-a100-11ea-9b44-36e998ee2e48.png)

Given that the only PRs are from dependabot, I doubt this will cause any problems. Render should still be able to access what it needs. Do you want to do the honors or should I?
